trello_utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import requests
+
+TRELLO_KEY = os.environ.get('TRELLO_KEY')
+TRELLO_TKN = os.environ.get('TRELLO_TKN')
+
+TRELLO_BOARD = '5f48fa1737fa77134ffe1e17'
+TRELLO_TODO_LIST = '5f48fa17b70abe80f7f0942d'
+TRELLO_DONE_LIST = '5f48fa17d5f15e744c86a725'
+TRELLO_URL_BASE = 'https://api.trello.com/1/'
+
+CARD_DONE_STATUS = 'Done!'
+CARD_TODO_STATUS = 'To-do'
+
+LOCAL_CARDS = []
+
+
+def trimCardsList(trello_cards):
+    for card in trello_cards:
+        updateOrAddCard(Card(
+            card['id'],
+            card['name'],
+            getCardStatus(card['id'])
+        ))
```

Comment:
First of all, why are you sharing a LOCAL_CARDS variable between here and updateOrAddCard - it's not obvious from reading this line that LOCAL_CARDS is being updated.
It would be much neater if updateOrAddCard took a card and a list as parameters and returned the list at the end of the function.

But actually, that's all unnecessary.

This function should be:
```
    local_cards = []
    for card in trello_cards:
        local_cards.append(Card(
            card['id'],
            card['name'],
            getCardStatus(card['id'])
        ))
    local_cards.sort(key=cardComparator)
    return local_cards
```